src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's a good practice to use classes instead of tag names for styling to avoid applying styles globally which can lead to unexpected results in larger projects. Consider adding a class to the body if you need to apply specific styles.

Reply:
its body without class